ql/src/java/org/apache/hadoop/hive/ql/plan/ImportTableDesc.java

```diff
@@ -332,4 +330,13 @@ public String getDatabaseName() {
   public boolean isMaterializedView() {
     return table.isMaterializedView();
   }
+
+  public TableType tableType() {
```

Comment:
do we really need the differentiation between materialized_view and view can we not just report state based on table / view.